### PR TITLE
Set installation env vars at install

### LINKED ISSUE #186: Launch kolibri with appropriate installer type env var set
In line with https://github.com/learningequality/kolibri/issues/7957 the batch file for launching Kolibri should properly set `KOLIBRI_INSTALLATION_TYPE` to `Windows`

### PR DESCRIPTION
**Pull request by rtibbles**
* Update inno setup to set installer env vars.
* Update readme.

Fixes #186

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,16 +12,17 @@ These are the steps how to do releases for the Kolibri Windows installer.  Note
 
 We are releasing `v1.3.1` as an example here.
 
-1. Tag the `master` branch as `v1.3.1` at `kolibri-installer-windows`
+1. Set the new version in KolibriSetupScript.iss where we set the KOLIBRI_INSTALLER_VERSION env var.
+2. Tag the `develop` branch as `v1.3.1` at `kolibri-installer-windows`
```
Comment (indirectlylit) [inline README.md:16]: if we are tagging `develop`, we might want to rename it to `main` and delete `master`

File: src/installer-source/KolibriSetupScript.iss [hunk 2]
```diff
@@ -645,6 +645,36 @@ begin
...
+            'KOLIBRI_INSTALLATION_TYPE',
+            'windows'
+        );
+
+        RegWriteStringValue(
+            HKLM,
+            'System\CurrentControlSet\Control\Session Manager\Environment',
+            'KOLIBRI_INSTALLER_VERSION',
+            'v1.5.0'
+        );
```
Comment (indirectlylit) [inline src/installer-source/KolibriSetupScript.iss:676]: main thing I'd be interested in making sure of is that these variables are accessible after installation, not just during the setup script run

**Review by indirectlylit — COMMENTED** [review]
Haven't tested and am not familiar with the syntax, but the update looks at least consistent with other parts of the script. If it works that's the important thing!

One other note is that we do have an uninstall script and might be good to add these to the cleanup:

https://github.com/learningequality/kolibri-installer-windows/blob/c469b7e602bb3a9cd6e272f7c2fe032e4951c5b8/src/installer-source/KolibriSetupScript.iss#L749-L765

File: src/installer-source/KolibriSetupScript.iss (hunk 2 above)
Comment (rtibbles) [reply src/installer-source/KolibriSetupScript.iss:676]: The other env vars referenced in here are used at run time, so my assumption was yes.

Comment (rtibbles) [reply src/installer-source/KolibriSetupScript.iss:676]: See here for example: https://github.com/learningequality/kolibri-installer-windows/blob/a8ea0684b38131bc60cacd5cbfb8830bd2c8a2ff/src/gui-source/Kolibri/Kolibri.cpp#L99

**Review by indirectlylit — APPROVED** [review]
thank you!

**Pull request merged**